coda-prototype/initUI.js

```diff
@@ -445,21 +445,19 @@ function initUI(dataset) {
 
     $("#export-dataset").click(() => FileUtils.saveDataset(newDataset));
 
-    $("#dataset-file").on("change", event => { // Fires when the dataset file has been changed by the file picker UI
+    /**
+     * Handles loading a new dataset after the user has selected a new dataset file via the Dataset dropdown.
```

Comment:
Could be shortened to "Loads a new dataset after the user has selected the file via the dataset dropdown"

Reply:
I disagree - one can choose either a scheme file or a dataset file from the Dataset dropdown.

The way to fix this properly would be to rename the Dataset dropdown - "File"?

Reply:
Made issue #86 for this.